refarch-integrations/refarch-dms-integration/refarch-dms-integration-fabasoft/refarch-dms-integration-fabasoft-mock/src/main/java/de/muenchen/refarch/integration/dms/fabasoft/mock/WiremockWsdlUtility.java

```diff
@@ -22,28 +22,30 @@
 import org.apache.commons.lang3.StringUtils;
 import org.w3c.dom.Document;
 
-public class WiremockWsdlUtility {
+public final class WiremockWsdlUtility {
+    private WiremockWsdlUtility() {
+    }
```

Comment:
Should we add throwing a Exception here as instantiating (even inside this class) is not allowed?

Reply:
see above